app/client/ui/ShareMenu.ts

```diff
@@ -277,7 +282,13 @@ function menuExports(doc: Document, pageModel: DocPageModel) {
       menuItem(() => gristDoc.app.comm.showItemInFolder(doc.name),
         t("Show in folder"), testId('tb-share-option')) :
         menuItem(() => downloadDocModal(doc, pageModel),
-        menuIcon('Download'), t("Download..."), testId('tb-share-option'))
+        menuIcon('Download'), t("Download document..."), testId('tb-share-option'))
+    ),
+    menuItem(
+      () => downloadAttachmentsModal(doc, pageModel),
+      menuIcon('Download'),
+      t('Download attachments...'),
+      testId('tb-share-option'),
```

Comment:
Should we call this testId in some other way?

Reply:
It's convention for everything in this list to have the same testId right now.

I'm happy to fix that - but it needs to be a separate PR or two, as SaaS uses these testIds too.